types/nft.ts

```diff
@@ -0,0 +1,12 @@
+export interface INFT {
+  name: string;
+  symbol: string;
+  description: string;
+  seller_fee_basis_points: number;
+  image: string;
+  external_url: string;
+  edition: number;
+  attributes: Record<string, string>[];
+  collection: Record<string, string>;
+  properties: Record<string, any>;
+}
```

Comment:
I think this type should be available in Metaplex, please try to find it there first